Sources/DequeModule/Deque._Storage.swift

```diff
@@ -212,3 +212,11 @@ extension Deque._Storage {
     }
   }
 }
+
+extension Deque._Storage {
+  @inlinable
+  @inline(__always)
```

Comment:
@lorentey Do these `inline` attributes look appropriate here?

Reply:
This one looks fine to me! This package generally wants all generic code to be `@inlinable` to allow specialization (and to prevent having to materialize metatypes just to call this function). (This may not remain a tenable approach for long, but as long as these constructs are distributed as a package, the drawbacks of exposing function bodies are less pressing than e.g. if they shipped in the stdlib.)

`@inline(__always)` encourages the optimizer to actually inline the body more often into clients. In the case of `==`, I expect this doesn't matter much either way.